docs/developer_guide/index.md

```diff
@@ -6,4 +6,6 @@
 2.  [Getting started](./getting_started.md)
 3.  [Using Ansible playbooks](./playbooks.md)
 4.  [Working with routes](./routes_aliases.md)
-5.  [Hello customer](./hello.md)
+5.  [Handling secrets](./secrets.md)
+6.  [Hello customer](./hello.md)
+7.  [Continous_Integration](./Continous_Integration.md)
```

Comment:
* `Continous_Integration` is not for humans :p
* also don't forget to fix the path once the file has been renamed